frontend/src/components/TransactionItem.tsx

```diff
@@ -243,6 +262,15 @@ function TransactionItem({ tx }: Props) {
                 </p>
               </div>
             )}
+            {tx.metadata?.comment && (
+              <div className="mt-6">
+                <p>Comment</p>
+                <p className="text-muted-foreground break-all">
+                  {tx.metadata?.comment}
+                </p>
+              </div>
+            )}
+            {/* We don't include zap request content as that is set as the description itself by the rails app */}
```

Comment:
This comment took me a minute to understand after reading it. I think here we should refer to "the rails app" as "the getalby.com backend". Also not sure we should even say it because hub runners could use a different lightning address server. But we could say "for Alby lightning addresses the description is automatically extracted from the zap request, and already displayed above"

Reply:
I was also thinking about it, that sounds good, changed!